### ISSUE TITLE
[Bug] Macros Change of several weapon /shield /libram (BB #675)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#675]
This issue was migrated from bitbucket.
**Original Reporter:** Alex_Step
**Original Date:** 27.08.2014 12:31:57 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/675

<hr>

The bug is the inability to change a few weapons in combat by clicking on the first macro.

For Example macros:

```
#!c++
#showtooltip Crusader Strike
/equip Brutal Gladiator's Bonegrinder
/equip Libram of Righteous Power
/startattack
/cast Crusader Strike

```

Should be in combat changed by all items + used spell

Now: In combat change by only one item - then have to press the macro again to load it.

This greatly affects pvp..

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Alex_Step
**Original Date:** 27.08.2014 12:34:11 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/675/#comment-12015747

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Alex_Step
**Original Date:** 27.08.2014 17:26:13 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/675/#comment-12022354

<hr>

The bug originated from over commit: https://bitbucket.org/oregon/oregoncore/commits/ae1b47c0d0614dda3f506236adb5ad132b387664

Without it, everything is normal.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 05.10.2014 04:56:42 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/675/#comment-12777561

<hr>

[Core/Players] Fix issue #675. Relics should NOT trigger GCD.

→ <<cset 5d515ab5388e>>
